modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/tree/BPlusTree.java

```diff
@@ -99,6 +100,8 @@
     /** Wrapper for tree pages operations. Noop by default. Override for test purposes. */
     public static volatile PageHandlerWrapper<Result> pageHndWrapper = (tree, hnd) -> hnd;
 
+    public static volatile IgniteRunnable destroyClosure = null;
```

Comment:
Need informative comment here.

Reply:
i think a little more - bad idea to use this additional closure for test purposes only here, u can override acquirePage for example.

Reply:
I tried to improve it by adding static h2TreeFactory field to H2TreeIndex, it will be more common.